src/client.ts

```diff
@@ -53,6 +53,10 @@ export class KolibriClient {
         this.delegate.addOnUserNotifyListener(listener);
     }
 
+    public addOnNodeNotifyListener(listener: (data: any[]) => void) {
```

Comment:
Why is this an array?
Shouldnt it be the same as nodeNotifyRequest.params on line 274 in base_client.ts ?

Reply:
Done, changed to a NodeNotifyObject

Reply:
done